.github/workflows/tox.yml

```diff
@@ -1,146 +1,72 @@
 name: tox
 
 on:
-  create:  # is used for publishing to TestPyPI
-    tags:  # any tag regardless of its name, no branches
+  push:
```

Comment:
This is not desired because it will double the number of builds when pushing to development branches.

For push, we want to build only on pushes to default branch. The rest are caused by pull_requests.